pyatlan/model/assets/process.py

```diff
@@ -212,6 +227,9 @@ class Attributes(Asset.Attributes):
         airflow_tasks: Optional[List[AirflowTask]] = Field(
             default=None, description=""
         )  # relationship
+        alteryx_workflow: Optional[AlteryxWorkflow] = Field(
+            default=None, description=""
+        )  # relationship
```

Comment:
This is a one-off customization (in-progress experiment) — we should remove from generated code.